lib/web_ui/dev/driver_version.yaml

```diff
@@ -1,4 +1,10 @@
-
+## Version to be picked by this PR. The LUCI will have multiple cipd packages
+## insalled in the recipe at the same time.
```

Comment:
> The LUCI will have multiple cipd packages insalled in the recipe at the same time.

This sentence needs more clarity. Which CIPD packages is this referring to? What do they have to do with drivers? What's significant about having multiple of them?

nit: s/insalled/installed/

Reply:
CIPD directories are visible to only a limited group of people so I'm not sharing those details here. But I can add some more details.

Reply:
I also documentation to README.